src/HttpOverStream.Server.Owin/CustomListenerHost.cs

```diff
@@ -22,13 +21,13 @@ public class CustomListenerHost : IDisposable
         private readonly Func<IDictionary<string, object>, Task> _app;
         private readonly ILogger _logger;
         private bool _disposed;
-        static byte[] _eol = Encoding.ASCII.GetBytes("\n");
+        static readonly byte[] _eol = Encoding.ASCII.GetBytes("\n");
 
         private CustomListenerHost(IListen listener, Func<IDictionary<string, object>, Task> app, IAppBuilder builder)
         {
             _listener = listener;
             _app = app;
-            _logger = builder.CreateLogger<CustomListenerHost>();
+            _logger = builder.CreateLogger("HttpOS.Owin.CLH");
```

Comment:
Would `HttpOverStream` work? 🤔

Reply:
What just HttpOverStream? It doesnt tell you which class the message came from, but maybe thats fine?

Reply:
Or not, that's probably fine as it is. It's just `HttpOS.Owin.CLH` is a little difficult to say out loud 😬

Reply:
I agree, its ugly. However, the most important thing is that we know without doubt where a log line originated from. These are for making diagnostics as specific as possible, not for readability. It wastes a huge amount of time if you have a debug issue but you're not sure where the error actually came from and you need to tweak the logs and rebuild. Hopefully no-one will need to say it out loud...